OpenQA/Qemu/Proc.pm

```diff
@@ -233,7 +236,8 @@ sub configure_blockdevs {
     my $is_first = 1;
     for my $k (sort grep { /^ISO_\d+$/ } keys %$vars) {
         my $addoniso = $vars->{$k};
-        my $i        = $k;
+        $addoniso = File::Spec->rel2abs($addoniso);
```

Comment:
Maybe assign this directly, e.g. `my $addoniso = File::Spec->rel2abs($vars->{$k});`.